src/mplutils.h

```diff
@@ -29,9 +29,14 @@
 #endif
 
 
+inline int mpl_round_to_int(double v)
+{
+    return (int)(v + ((v >= 0.0) ? 0.5 : -0.5));
+}
+
 inline double mpl_round(double v)
 {
-    return (double)(int)(v + ((v >= 0.0) ? 0.5 : -0.5));
+    return (double)mpl_round_to_int(v);
```

Comment:
Is there a reason `mpl_round` doesn't work here?  Does `mpl_round` handle 0.5 differently?

Reply:
`mpl_round` does `(double)(int)...` while `mpl_round_to_int` does `(int)...`. So I thought that inlining it will avoid code duplication. It should be identical.

Reply:
Oh duh, this is `mpl_round` :sheep:!